response_play.go

```diff
@@ -0,0 +1,135 @@
+	Summary            string          `json:"summary,omitempty"`
+	Self               string          `json:"self,omitempty"`
+	HTMLURL            string          `json:"html_url,omitempty"`
+	Name               string          `json:"name,omitempty"`
+	Description        string          `json:"description"`
+	Team               *APIReference   `json:"team,omitempty"`
+	Subscribers        []*APIReference `json:"subscribers,omitempty"`
+	SubscribersMessage string          `json:"subscribers_message"`
+	Responders         []*APIReference `json:"responders,omitempty"`
+	RespondersMessage  string          `json:"responders_message"`
+	Runnability        *string         `json:"runnability"`
+	ConferenceNumber   *string         `json:"conference_number"`
+	ConferenceUrl      *string         `json:"conference_url"`
+	ConferenceType     *string         `json:"conference_type"`
+}
+
+type ListResponsePlaysResponse struct {
+	ResponsePlays []ResponsePlay `json:"response_plays"`
+}
+
+type ListResponsePlaysOptions struct {
+	FilterForManualRun bool   `url:"filter_for_manual_run,omitempty"`
+	Query              string `url:"query,omitempty"`
+}
+
+// ListResponsePlays lists existing response plays.
+func (c *Client) ListResponsePlays(ctx context.Context, o ListResponsePlaysOptions) (ListResponsePlaysResponse, error) {
+	var result ListResponsePlaysResponse
+
+	v, err := query.Values(o)
+	if err != nil {
+		return result, err
+	}
+
+	resp, err := c.get(ctx, "/response_plays?"+v.Encode())
+	if err != nil {
+		return result, err
+	}
+
+	if err = c.decodeJSON(resp, &result); err != nil {
+		return result, err
+	}
+
+	return result, nil
+}
+
+// CreateResponsePlay creates a new response play.
+func (c *Client) CreateResponsePlay(ctx context.Context, rp ResponsePlay) (ResponsePlay, error) {
+	d := map[string]ResponsePlay{
+		"response_play": rp,
+	}
+
+	resp, err := c.post(ctx, "/response_plays", d, nil)
+	return getResponsePlayFromResponse(c, resp, err)
+}
+
+// GetResponsePlay gets details about an existing response play.
+func (c *Client) GetResponsePlay(ctx context.Context, id string) (ResponsePlay, error) {
+	resp, err := c.get(ctx, "/response_plays/"+id)
+	return getResponsePlayFromResponse(c, resp, err)
+}
+
+// UpdateResponsePlay updates an existing response play.
+func (c *Client) UpdateResponsePlay(ctx context.Context, rp ResponsePlay) (ResponsePlay, error) {
+	d := map[string]ResponsePlay{
+		"response_play": rp,
+	}
+
+	resp, err := c.put(ctx, "/response_plays/"+rp.ID, d, nil)
+	return getResponsePlayFromResponse(c, resp, err)
+}
+
+// DeleteResponsePlay deletes an existing response play.
+func (c *Client) DeleteResponsePlay(ctx context.Context, id string) error {
+	_, err := c.delete(ctx, "/response_plays/"+id)
+	return err
+}
+
+// RunResponsePlay runs a response play on a given incident.
+func (c *Client) RunResponsePlay(ctx context.Context, rp string, incident Incident, from string) error {
```

Comment:
Sorry for not explicitly calling this in an earlier review. I think it also makes sense to make this an `incidentID string` instead of `incident Incident`.

I think we could also make it a little more self-documenting, for those that use something like LSP to overlay method signatures as they are writing them. I'm thinking:

```Go
func (c *Client) RunResponsePlay(ctx context.Context, responsePlayID string, incidentID string, from string) error {
```

Reply:
no problem - I'll get this fixed up

Reply:
thanks for the clarification @theckman  - I have pushed a commit to change the signature :)